package.json

```diff
@@ -116,7 +118,8 @@
     "storybook:deploy": "storybook-to-ghpages '--script=storybook:build'",
     "semantic-release": "semantic-release",
     "stylelint": "stylelint --fix 'sass/**/*.scss'",
-    "travis-deploy-once": "travis-deploy-once"
+    "travis-deploy-once": "travis-deploy-once",
+    "openurl": "node -p 'require(\"openurl\").open(\"http://localhost:6006\")'"
```

Comment:
Would it make sense to keep it inside the `storybook` context?
e.g. `npm run storybook:openurl`

Reply:
It *is* only being used by storybook, I have no issues with renaming it.